storage/src/bytes.rs

```diff
@@ -0,0 +1,312 @@
+use std::path::Path;
+use std::path::PathBuf;
+
+use bytes::Buf;
+use bytes::BytesMut;
+use reqwest::Response;
+use tokio::fs::File;
+use tokio::io::AsyncReadExt;
+
+use super::{Error, Result};
+pub use bytes::Bytes;
+
+#[derive(Debug)]
+pub struct BytesReader {
```

Comment:
Note this file was manually written to be a helper struct for working with file uploads and storage downloads. It is not generated.